android/src/main/java/com/reactnativemembrane/MembraneModule.kt

```diff
@@ -440,6 +442,20 @@ class MembraneModule(reactContext: ReactApplicationContext) :
   @ReactMethod
   fun removeListeners(count: Int?) {}
 
+  @ReactMethod
+  fun changeWebRTCLoggingSeverity(severity: String) {
```

Comment:
add promise, just like in other react methods

Reply:
Done.